alchemy/src/cloudflare/bound.ts

```diff
@@ -16,6 +16,7 @@ import type { VectorizeIndexResource as _VectorizeIndex } from "./vectorize-inde
 import type { Worker as _Worker } from "./worker.ts";
 import type { Workflow as _Workflow } from "./workflow.ts";
 import type { Images as _Images } from "./images.ts";
+import type { ImagesBinding } from "@cloudflare/workers-types";
```

Comment:
you don't need the import - these types are globally available, see how the rest of the file doesn't need it?